report-viewer/src/model/factories/OverviewFactory.ts

```diff
@@ -4,11 +4,11 @@ import { ComparisonListElement } from "../ComparisonListElement";
 import { Cluster } from "@/model/Cluster";
 import store from "@/store/store";
 import { Version } from "../Version";
+import versionJson from "../../version.json";
 
 export class OverviewFactory {
 
-  // TODO: Find a better more visible position to store this
-  static reportViewerVersion: Version = {major: 4, minor: 0, patch: 0};
+  static reportViewerVersion: Version = versionJson["report_viewer_version"] !== undefined ? JSON.parse(JSON.stringify(versionJson["report_viewer_version"])) : {major: -1, minor: -1, patch: -1};
```

Comment:
Why do you stringify and parse here? Can't you access the properties of the `versionJson["report_viewer_version"]` directly?

Reply:
@sebinside it can access directly right now.